src/main/java/org/apache/commons/io/output/FileWriterWithEncoding.java

```diff
@@ -46,9 +46,53 @@
  *
  * @since 1.4
  */
-public class FileWriterWithEncoding extends Writer {
-    // Cannot extend ProxyWriter, as requires writer to be
-    // known when super() is called
+public class FileWriterWithEncoding extends ProxyWriter {
+
+    private static final Writer NULL_WRITER = new Writer() {
```

Comment:
Not sure if it's handling `null`s, or being used when some value is `null`. But maybe `NOOP_WRITER` would be a better name?

Reply:
Just saw your comment about the `Writer.nullWriter()`, I guess that's where you got the name from :+1: no need to change it.